lms/djangoapps/courseware/toggles.py

```diff
@@ -153,3 +166,9 @@ def course_is_invitation_only(courselike) -> bool:
 
 def learning_assistant_is_active(course_key):
     return COURSEWARE_LEARNING_ASSISTANT.is_enabled(course_key)
+
+def courseware_mfe_search_enabled():
```

Comment:
Why don't we think about course_waffle_override at this moment?
I believe it would be very likely we want to release by course, not by number of learners, right?

Reply:
We discussed it with @ashultz0 and since the requirement was for staff only, the `WaffleFlag` at this stage was more convenient, since Course waffle flags don't have user distinction (it's either enabled or disabled for the course). We did however included the course ID on the URL. I believe in the future when the feature is on a safe state for students to use we can switch to course waffles.

Reply:
my understanding was we wanted all courses, just edx staff, which is easiest to achieve with waffle flag

Reply:
I see. Thank you for the context